app/src/main/java/com/example/appmarvel/services/serviceinterface/GetCharactersListService.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.appmarvel.services.serviceinterface
+
+import com.example.appmarvel.entity.Character
+import com.example.appmarvel.services.utils.Result
+
+interface GetCharactersListService {
```

Comment:
the name should be something like "MarverService" as you can use it for any endpoint on this api

Reply:
Done